VariablesHomework/VariablesHomework.cs

```diff
@@ -16,130 +22,131 @@ public void Declaration()
 
-            float f = 0.0f;
-            Assert.IsTrue(f == 3.5); // Make this pass without changing this statement (except you might need to add something)
-
+            float f = 3.5f;
+        Assert.IsTrue(f == 3.5); // Make this pass without changing this statement (except you might need to add something)
+            f = 99.9f;
             Assert.IsTrue(f == 99.9); // Then make this pass without changing this statement (except you might need to add something)
 
             bool b = false;
-            Assert.IsTrue(b); // Make this pass without changing this statement
-
+        Assert.IsTrue(b); // Make this pass without changing this statement
+            b = false;
             Assert.IsTrue(b == false); // Then make this pass without changing this statement
 
-            double d = 0.0;
-            Assert.IsTrue(d == 10.0); // Make this pass without changing this statement
-
+            double d = 10.0d;
+        Assert.IsTrue(d == 10.0); // Make this pass without changing this statement
+            d = 20.0;
             Assert.IsTrue(d == 20.0); // Then make this pass without changing this statement
 
 
-            string s = "";
-            Assert.IsTrue(s == "word"); // Make this pass without changing this statement
-
+            string s = "word";
+        Assert.IsTrue(s == "word"); // Make this pass without changing this statement
+            s = "next";
             Assert.IsTrue(s == "next"); // Then make this pass without changing this statement
 
-            char c = 'a';
-            Assert.IsTrue(c == 'b'); // Make this pass without changing this statement
 
+            char c = 'b';
+        Assert.IsTrue(c == 'b'); // Make this pass without changing this statement
+            c = 'c';
             Assert.IsTrue(c == 'c'); // Then make this pass without changing this statement
 
             int z = i;
-            Assert.IsTrue(z == /*Answer here*/); // Make this pass by ONLY filling in the answer
+        Assert.IsTrue(z == i); // Make this pass by ONLY filling in the answer
         }
 
         // Refer to this if you need help https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators
         [TestMethod]
         public void Operators()
         {
             int a = 0;
-            int b = a + 13; 
-            Assert.IsTrue(b == /*Answer here*/); // Make this pass by ONLY filling in the answer
+            int b = a + 13;
+        Assert.IsTrue(b == 13); // Make this pass by ONLY filling in the answer
 
             float f = 12.3f;
-            float c = f - 7.43f;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass by ONLY filling in the answer
-            Assert.IsTrue(f == /*Answer here*/); // Make this pass by ONLY filling in the answer
+        float c = f - 7.43f;
+        Assert.IsTrue(c == 4.87); // Make this pass by ONLY filling in the answer
```

Comment:
Remember how floating point math can be "fuzzy"?
The answer here won't be exactly 4.87. And it will need a suffix to treat it as a float (rather than as a double).